packages/header-checker-lint/src/header-parser.ts

```diff
@@ -20,7 +20,7 @@ export interface LicenseHeader {
   year?: number;
 }
 
-const COPYRIGHT_REGEX = /\s*([*#]|\/\/) \s*Copyright (\d{4}(-\d{4})?) ([\w\s,]+)\.?/;
+const COPYRIGHT_REGEX = /\s*(^|[*#]|\/\/) ?\s*Copyright (\d{4}(-\d{4})?) ([\w\s,]+)\.?/;
```

Comment:
I think the caret only mean the start of the line at the start of the regex. We might be able to accomplish this with:
```suggestion
const COPYRIGHT_REGEX = /^\s*([*#]|\/\/)\s*Copyright (\d{4}(-\d{4})?) ([\w\s,]+)\.?/;
```

The 0+ matcher on `\s` should capture the optional space character and we're probably fine making sure we're matching the start of the line.

Reply:
Checking out & trying this locally, it looks like this may break the html test